nodes/src/nodes/memory_persistent/memory_store.py

```
@@ -0,0 +1,710 @@
+            pipe.execute()
+            # Preserve TTL alignment with session metadata
+            remaining_ms = self._client.pttl(self._meta_key(session_id))
+            if remaining_ms and remaining_ms > 0:
+                self._client.pexpire(history_key, remaining_ms)
+
+    def close(self) -> None:
+        """Close the Redis connection explicitly."""
+        try:
+            self._client.close()
+        except Exception:
+            pass
```

Ruff (0.15.9)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)